collectionApp.css

```diff
@@ -49,8 +86,11 @@ h1 {
     }
 
     .textContainer {
-        flex-container: row;
         padding: 10px;
         align-content: center;
```

Comment:
This also needs `display: flex` in the mobile version above so that `align-content` will work too